platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -494,11 +494,12 @@ namespace platoon_strategic_ihp
                 status_msg.host_platoon_position = pm_.getNumberOfVehicleInFront();
                 ROS_DEBUG_STREAM("pm platoonsize: " << pm_.getTotalPlatooningSize() << ", platoon_leader " << platoon_leader.staticId);
 
+                // TODO: To uncomment the following lines, platooninfo msg must be updated
```

Comment:
Is this message fix just not a priority right now?

Reply:
It should be updated soon, before trying to test the IHP control plugin.